ta/pkcs11/src/pkcs11_helpers.h

```diff
@@ -6,11 +6,26 @@
 #ifndef PKCS11_HELPERS_H
 #define PKCS11_HELPERS_H
 
+#include <pkcs11_ta.h>
 #include <stdint.h>
 #include <stddef.h>
 
+/* Short aliases for return code */
```

Comment:
These doesn't seem very useful. You save a few keystrokes when typing, but it doesn't help when reading.

Reply:
ok